packages/flutter/test/widgets/tap_region_test.dart

```diff
@@ -1177,4 +1177,69 @@ void main() {
     expect(count1, 1); // tapRegion1 should respond.
     expect(count2, 1); // tapRegion2 should not respond anymore.
   });
+
+  // Regression test for the consumeOutsideTaps issue when navigating between pages
```

Comment:
Just leave the GitHub URL: "Regression test for https://github.com/flutter/flutter/issues/169061."